### ISSUE TITLE
[require-hook] False positive on non-test file

### ISSUE THREAD

**Issue by aryzing** [jest-community/eslint-plugin-jest#976]
I have a file in `src/utils/bluebird.ts` that for some reason is triggering the `require-hook` rule :shrug: 

```ts
import Bluebird from "bluebird";

Bluebird.config({
  longStackTraces: true,
  cancellation: true,
  warnings: true,
});

export default Bluebird;
```

Error:

```
/home/aryzing/workspace/project/src/utils/bluebird.ts
  3:1  error  This should be done within a hook  jest/require-hook
```

**Comment by G-Rath** [comment]
Duplicate of #934
